### DISCUSSION TITLE
Grunge Stack Architecture

### DISCUSSION THREAD

**Discussion by justindra** [remix-run/remix#3328] [Q&A]
Hi,

Wanted to ask if the grunge stack example currently has any sort of CDN caching? Trying to understand what architect is doing. It looks like it is just using a normal Lambda function to serve the files as required and not Lambda@Edge?

Basically just trying to re-create it using SST (or AWS CDK) as the rest of my infrastructure is already defined there and don't want to use yet another tool.

Cheers!

**Comment by fezproof** [comment]
Looking at how architect works, there are no cache headers added to the static files. See https://arc.codes/docs/en/guides/frontend/static-assets#fingerprinted-file-paths. There is also no configuration for cloud-front at all.

**Comment by justindra** [comment]
Work is currently being done over at SST to support this using a Construct.
https://github.com/serverless-stack/serverless-stack/pull/1800
